.github/workflows/e2e-tests.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       uses: ./.github/actions/prepare-uberjar-artifact
 
   e2e-tests:
-    runs-on: buildjet-2vcpu-ubuntu-2004
+    runs-on: buildjet-4vcpu-ubuntu-2004
```

Comment:
@ariya shouldn't we use the 8vcpu which proved to be the best on overall buildjet machines for the e2e ?

Reply:
Let's do a baby step since bumping the speed a bit already reveals more flakiness due to race conditions.